src/pfe/portal/modules/User.js

```diff
@@ -639,54 +639,51 @@ module.exports = class User {
   /**
    * Function to write workspace settings
    */
-  async writeWorkspaceSettings(workspaceSettings) {
+  async writeWorkspaceSettings(address, namespace) {
     let retval;
     try {
       log.info(`Writing workspace settings file.`);
-      retval = await this.fw.writeWorkspaceSettings(workspaceSettings);
+      retval = await this.fw.writeWorkspaceSettings(address, namespace);
     } catch (err) {
       log.error(`Error in writeWorkspaceSettings`);
-      log.error(err);
+      throw err;
```

Comment:
Same as above, do we need to log the contents of the err object here?

Reply:
Error is logged in `imagePushRegistry.route.js`

```
 retval = await user.writeWorkspaceSettings(address, namespace);
    }
    res.status(retval.statusCode).send(retval);
  } catch (error) {
    log.error(error);
    const workspaceSettings = {
      statusCode: 500,
      imagePushRegistryTest: false,
      msg: error.message
    }
    res.status(500).send(workspaceSettings);
  }
```

Didnt want multiple places to have same err being logged